bin/fin

```diff
@@ -119,10 +119,33 @@ is_katacoda ()
 	[[ "$KATACODA" != "" ]]
 }
 
-is_github_actions () {
+is_github_actions ()
+{
 	[[ "$GITHUB_RUN_ID" != "" ]]
 }
 
+# Returns x86_64 / aarch64 (aka "old school")
+host_arch () {
+	case "$(uname -m)" in
+		x86_64) echo "x86_64" ;;
+		amd64) echo "x86_64" ;;
+		aarch64) echo "aarch64" ;;
+		arm64) echo "aarch64" ;;
+		* ) false;;
```

Comment:
This case will lead to issues in URLs. We should default to something e.g. x86_64